conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -128,6 +129,20 @@ def paginated_api_response(
     }
 
 
+def deprecated(sunset_date):
```

Comment:
It needs docstrings, it would be particularly helpful to document the expected format of the `sunset_date`)